jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBPreparedStatement.java

```diff
@@ -181,53 +181,63 @@ public void setArray(int parameterIndex, Array x) throws SQLException {
 
   @Override
   public void setAsciiStream(int parameterIndex, InputStream x) throws SQLException {
-    throw new SQLException(METHOD_NOT_SUPPORTED_STRING);
+   // throw new SQLException(METHOD_NOT_SUPPORTED_STRING);
+	  throw new SQLException(Constant.PARAMETER_SUPPORTED);
   }
 
   @Override
   public void setAsciiStream(int parameterIndex, InputStream x, int length) throws SQLException {
-    throw new SQLException(METHOD_NOT_SUPPORTED_STRING);
+   // throw new SQLException(METHOD_NOT_SUPPORTED_STRING);
```

Comment:
Use `2 spaces` instead of `TAB` as indent characters.
Consider importing the code format  `$IoTDB_HOME/java-google-style.xml` into IDE.